kolibri/plugins/pdf_viewer/assets/src/views/PdfRendererIndex.vue

```diff
@@ -151,9 +147,28 @@
         }
         return 0;
       },
+      savedVisitedPages: {
+        get() {
+          if (this.extraFields && this.extraFields.contentState) {
+            return this.extraFields.contentState.savedVisitedPages || {};
+          }
+          return {};
+        },
+        set(value) {
+          this.visitedPages = value;
+        },
+      },
       fullscreenText() {
         return this.isInFullscreen ? this.$tr('exitFullscreen') : this.$tr('enterFullscreen');
       },
+      /* eslint-disable kolibri/vue-no-unused-properties */
+      /**
+       * @public
+       */
+      defaultDuration() {
+        return this.targetTime;
```

Comment:
Is `targetTime` used anywhere else? Like in the EPub renderer we may be able to just rename it to `defaultDuration`.